From 5072571328ee501175ba746e872813c0579c167b Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Thu, 13 Mar 2025 09:24:23 -0400
Subject: [PATCH 2/2] Remove Git commands (not useful in tarball)

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 cmake/flatbuffers.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/flatbuffers.cmake b/cmake/flatbuffers.cmake
index 69deed05..fb8969f4 100644
--- a/cmake/flatbuffers.cmake
+++ b/cmake/flatbuffers.cmake
@@ -64,6 +64,7 @@ function(vpux_add_flatc_target FLATC_TARGET_NAME)
 
 endfunction()
 
+# TODO: Deal with this usage of Git
 find_package(Git REQUIRED)
 function(vpux_gf_version_generate SRC_DIR DST_DIR)
 
-- 
2.50.0

