From 7e8da7802b6a310a659eb8e25917a1b68ffdc99e Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Wed, 25 Jun 2025 23:57:38 -0400
Subject: [PATCH 3/3] Replace nlohmann_json with Fedora's json-devel

---
 thirdparty/json/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thirdparty/json/CMakeLists.txt b/thirdparty/json/CMakeLists.txt
index 0e6fe3ad6a..fcb47304d9 100644
--- a/thirdparty/json/CMakeLists.txt
+++ b/thirdparty/json/CMakeLists.txt
@@ -4,4 +4,4 @@
 
 # suppress shadowing names warning
 set(JSON_SystemInclude ON CACHE BOOL "" FORCE)
-add_subdirectory(nlohmann_json EXCLUDE_FROM_ALL)
+#add_subdirectory(nlohmann_json EXCLUDE_FROM_ALL)
-- 
2.49.0

